app/controllers/course/statistics/answers_controller.rb

```diff
@@ -76,17 +76,17 @@ def question_index(question_id)
   end
 
   def fetch_all_answers(submission_id, question_id, limit)
-    @all_answers = if limit
+    @all_answers = if limit == -1
```

Comment:
Ok I see you've updated the query to adjust the `limit` parameter to take in a number. Please squash the relevant commits.

Reply:
Done